client/src/components/DisplayTable/DisplayTable.yaml

```diff
@@ -18,12 +18,12 @@ items_per_page_description:
   fr: Choisir le nombre d'éléments à afficher à la fois sur le tableau
 show_all:
   en: Show all
-  fr: TODO
+  fr: Tout afficher
```

Comment:
I think it's "Afficher tout", but I'm no authority.

Reply:
It's what Andre sent me 🤷

Reply:
I would also be inclined to say that it's "Afficher tout"